CMakeLists.txt

```
@@ -171,6 +171,7 @@ adios_option(Endian_Reverse "Enable support for Little/Big Endian Interoperabili
 adios_option(Sodium     "Enable support for Sodium for encryption" AUTO)
 adios_option(Catalyst   "Enable support for in situ visualization plugin using ParaView Catalyst" AUTO)
 adios_option(AWSSDK     "Enable support for S3 compatible storage using AWS SDK's S3 module" OFF)
+adios_option(Derived    "Enable support for derived variables" OFF)
```

```suggestion
adios_option(Derived_Variables    "Enable support for derived variables" OFF)
```

How about using a more verbose var name? just `derived` will be hard to be put in context for users.

---

Sure, that makes sense. Will update this PR with this change